chemicaltoolbox/xchem-deep/xchem_deep_score.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="xchem_pose_scoring" name="XChem pose scoring" version="0.1.0">
+    <description>- perform pose scoring using deep learning</description>
+
+    <requirements>
+        <!--requirement type="package" version="3.0.0">openbabel</requirement-->
+        <!--requirement type="package" version="3.7">python</requirement-->
+        <!-- many other requirements are needed -->
+        <container type="docker">informaticsmatters/deep-app-ubuntu-1604:latest</container>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    ln -s '$ligands' ligands.sdf &&
+    ln -s '$receptor' receptor.pdb &&
+    python '$__tool_directory__/xchem_deep_score.py' -i ligands.sdf -r receptor.pdb --mock &&
+    cp output.sdf $output
```

Comment:
Maybe you can change this line (https://github.com/bgruening/galaxytools/pull/951/files#diff-698fab259d30bbd91e699904c70e218aR132) so it doesn't force an `sdf` extension? Then you can just have `-o $output` and dispense with the `cp`.

Reply:
I cleaned it up slightly, but the results are written to the work_dir so it still has to be copied.

Reply:
Ah fair enough